support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -61,19 +61,26 @@ const getDisplayPrice = (
 	return `${currency}${fixDecimals(price)}/${billingPeriod}`;
 };
 
-const getDigitalEditionPrices = (countryGroupId: CountryGroupId): string => {
+const getDigitalEditionPrice = (
+	countryGroupId: CountryGroupId,
+	billingPeriod: BillingPeriod,
+): string => {
 	const currencyKey = detect(countryGroupId);
 	const currency = currencies[currencyKey].glyph;
 	const product = productCatalog['DigitalSubscription'];
-	const priceMonthly = product?.ratePlans[Monthly]?.pricing[currencyKey];
-	const currencyPriceMonthly = priceMonthly
-		? `${currency}${fixDecimals(priceMonthly)}/${Monthly}`
+	const price = product?.ratePlans[billingPeriod]?.pricing[currencyKey];
+	return price
+		? `${currency}${fixDecimals(price)}/${
+				billingPeriod === Annual ? 'Annually' : billingPeriod
+		  }`
```

Comment:
Probably for readability we could early return if not compute values and return. 
```suggestion
	const price = product?.ratePlans[billingPeriod]?.pricing[currencyKey];
	if (!price) { return null }
	const fixedPriceWtihCurrency = `${currency}${fixDecimals(price)}`;
	const billingFrequency = billingPeriod === Annual ? 'Annually' : billingPeriod;
	return `${fixedPriceWtihCurrency}/${billingFrequency}`
```

Reply:
Yes thats more read-able, thank, updated.